playbooks/provisioner/openstack/main.yaml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
```

Comment:
why are we calling both roles?
shouldn't there be a "when: neutron/nova" statement?
Personally, I would rather you do a role with: provisioner/openstack
that it's main has: "- include: {{ provisioner.network.component }}.yml"
Does it make sense?